src/plugins/data/public/ui/saved_query_management/saved_query_management_component.tsx

```diff
@@ -70,6 +69,7 @@ interface Props {
   onClearSavedQuery: () => void;
   closeMenuPopover: () => void;
   saveQuery: (savedQueryMeta: SavedQueryMeta, saveAsNew?: boolean) => Promise<void>;
+  saveQueryIsDisabled?: boolean;
```

Comment:
i'm confused how would saved query be disabled from the saved query management page?